src/crypto/CHIPCryptoPAL.h

```diff
@@ -1475,6 +1479,38 @@ class SymmetricKeyContext
     virtual void Release() = 0;
 };
 
+/**
+ *  @brief P256 key builder for Operatinal and Ephermal keys.
+ **/
+class P256KeypairBuilder
+{
+public:
+    virtual ~P256KeypairBuilder() {}
+
+    /**
+     * @brief Create the keypair for given fabric Index.
+     * @return Return reference to P256 Keypair
+     **/
+    virtual P256Keypair * BuildP256KeyPairForOperationalKey(uint64_t fabricIndex) = 0;
+
+    /**
+     * @brief Create the ephermal keypair for CASE.
+     * @return Return reference to P256 Keypair
```

Comment:
Please document how memory should be managed by the caller

Reply:
@sujaygkulkarni-nxp ?

Reply:
This still needs to be done: what do callers need to do to manage the resources involved?